R/zchunk_L251.en_ssp_nonco2.R

```diff
@@ -1,27 +1,32 @@
 #' module_emissions_L251.en_ssp_nonco2
 #'
-#' Briefly describe what this chunk does.
+#' This chunk produces regional non-CO2 emissions coefficient data for SSPs 1/5, 2, and 3/4 as well as a GDP control.
```

Comment:
I asked Cary to change those back. 1.5 reads like this is referring to a 1.5 scenario. In some of the text we did change this to be more explicit (e.g. SSP1 and SSP5), but to keep length down some continuation is useful in this particular line.